packages-exp/auth-exp/test/integration/webdriver/util/test_runner.ts

```diff
@@ -31,7 +31,7 @@ import { AuthDriver } from './auth_driver';
  */
 
 interface TempSuite {
-  generator: (driver: AuthDriver) => void;
+  generator: (driver: AuthDriver, browser?: string) => void;
```

Comment:
```suggestion
  generator: (driver: AuthDriver, browser: string) => void;
```

The question mark makes the argument optional for the _caller_. The implementation can always omit trailing parameters -- there is no need to do anything. `(a) => foo` is always assignable to `(a: AuthDriver, b: string) => void`.